uber/models.py

```diff
@@ -1071,9 +1078,12 @@ def _staffing_adjustments(self):
         if self.badge_type == c.STAFF_BADGE:
             self.staffing = True
 
+        # remove trusted status from any dept we are not assigned to
+        self.trusted_depts = ','.join([str(td) for td in self.trusted_depts_ints if td in self.assigned_depts_ints])
```

Comment:
@EliAndrewC this is the line that now removes you from being trusted in a dept which you are no longer assigned to